apmpackage/apm/0.1.0/manifest.yml

```
@@ -35,12 +35,84 @@ policy_templates:
         title: Secret token
         required: false
         show_user: true
+      - name: api_key_enabled
+        type: bool
+        title: Enable API Key Auth from APM Agents
+        required: false
+        show_user: true
+        default: false
       - name: enable_rum
         type: bool
         title: Enable RUM
         required: true
         show_user: true
         default: false
+      - name: api_key_limit
+        type: int
+        title: Maximum number of allowed API Keys
```

```suggestion
        title: Maximum number of allowed API Keys between APM Server and Agents
```

We'll possibly have more api keys in the config, better to be explicit